tools/SetupFlow/DevHome.SetupFlow/Views/SetupTargetReviewView.xaml

```diff
@@ -83,24 +84,16 @@
                             Grid.Column="2"
                             Margin="0 0 5 0"
                             x:Uid="SetupTargetReviewPageComputeSystemVersion"
+                            Visibility="{x:Bind ViewModel.OsName, Mode=OneWay, Converter={StaticResource EmptyObjectToObjectConverter}}"

```

Comment:
I could definitely be misremembering, but have you tested this with both scenarios? I have a vague memory of trying to do something like this and realizing that Collapsed still resolved to "true" or something like that.

Reply:
Hey yes, I tested both scenarios. The key here is making the `OsName = null`. In the second video at timestamp `0:13 seconds` that shows what happens when `OsName is null`. The version text box and the string next to it does not appear. Then in timestamp `0:27 seconds` is what happens when `OsName is not null`